### ISSUE TITLE
Unrecognized notarization status: "in progress" 

### ISSUE THREAD

**Issue by gungorbudak** [karaggeorge/electron-builder-notarize-pkg#2]
Hi,

I'm using this package to sign and notarize my pkg installer but following exception is thrown which is weird because the code specifically checks for the equality to `in progress` and recursively waits for the next time.

What could be the issue here?

Thanks

```
 electron-builder  version=22.6.0 os=19.4.0
  • loaded configuration  file=package.json ("build" field)
  • loaded parent configuration  file=/path/node_modules/electron-webpack/out/electron-builder.js
  • writing effective config  file=dist/builder-effective-config.yaml
  • packaging       platform=darwin arch=x64 electron=8.2.5 appOutDir=dist/mac
after build; disable sandbox
  • signing         file=dist/mac/app.app identityName=Developer ID Application: Name (ID) identityHash=some_hash provisioningProfile=none
  • building        target=pkg arch=x64 file=dist/installer.pkg
Signing /path/dist/installer.pkg...
Notarizing /path/dist/installer.pkg...
  ⨯ Unrecognized notarization status: "in progress"  stackTrace=
                                                       Error: Unrecognized notarization status: "in progress"
                                                           at waitForNotarize (/path/node_modules/electron-builder-notarize-pkg/index.js:186:11)
                                                           at processTicksAndRejections (internal/process/task_queues.js:97:5)
                                                           at waitForNotarize (/path/node_modules/electron-builder-notarize-pkg/index.js:174:5)
                                                           at waitForNotarize (/path/node_modules/electron-builder-notarize-pkg/index.js:174:5)
                                                           at waitForNotarize (/path/node_modules/electron-builder-notarize-pkg/index.js:174:5)
                                                           at waitForNotarize (/path/node_modules/electron-builder-notarize-pkg/index.js:174:5)
                                                           at module.exports (/path/node_modules/electron-builder-notarize-pkg/index.js:261:3)
                                                           at path/node_modules/app-builder-lib/src/index.ts:64:38
                                                           at executeFinally (/path/node_modules/builder-util/src/promise.ts:12:14)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by karaggeorge** [comment]
Hmm, that is weird. I'll have to check

**Comment by karaggeorge** [comment]
Found the issue: https://github.com/karaggeorge/electron-builder-notarize-pkg/commit/07f3197a27ff3eeb259d817fff3108bc1a8c55de

Released this and the other PR in `1.1.0`
